docker-compose.override.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  express-server:
+    image: ${DOCKERHUB_NAME}/${IMAGE_NAME}:${IMAGE_VERSION}
+    depends_on:
+      - postgres-primary-db
+    command: ['npm', 'run', 'dev']
+    container_name: ${IMAGE_NAME}
+    volumes:
+      - '.:/src/app:rw'
+    env_file: .env
+    environment:
```

Comment:
Don't need `environment` and `env_file`. Just stick with `env_file` and put everything in `.env`.